test/conftest.py

```diff
@@ -263,4 +263,4 @@ def git_urls():
 
 @pytest.fixture()
 def invalid_urls():
-    return []
+    return ['not a valid URL']
```

Comment:
Can we make this a url, one that the regex does not match?

Reply:
I can't think of one which might not be supported. May be better to rename this fixture to `invalid_strings` and the test to something like `test_parse_raises_on_invalid_string`?

Reply:
ya, renaming it sounds good